youtube_transcript_api/_api.py

```diff
@@ -104,7 +118,23 @@ def get_transcript(cls, video_id, languages=('en',), proxies=None):
         :type languages: list[str]
         :param proxies: a dictionary mapping of http and https proxies to be used for the network requests
         :type proxies: {'http': str, 'https': str} - http://docs.python-requests.org/en/master/user/advanced/#proxies
+        :param cookies: a string of the path to a text file containing youtube authorization cookies
+        :type cookies: str
         :return: a list of dictionaries containing the 'text', 'start' and 'duration' keys
         :rtype [{'text': str, 'start': float, 'end': float}]:
         """
-        return cls.list_transcripts(video_id, proxies).find_transcript(languages).fetch()
+        return cls.list_transcripts(video_id, proxies, cookies).find_transcript(languages).fetch()
+    
+    @classmethod
+    def load_cookies(cls, cookies):
+        cj = {}
+        try:
+            cj = cookiejar.MozillaCookieJar()
+            cj.load(cookies)
+        except IOError as e:
+            raise CookiePathInvalid
+        except FileNotFoundError as e:
+            raise CookiePathInvalid
+        if not cj:
+            raise CookiesInvalid
```

Comment:
I am not quite sure about this, in which case does this happen? Isn't this always truthy as `cj` always is a `MozillaCookieJar` object (unless an error occurred, in which case execution will be stopped before this line of code anyways). The `CAUSE_MESSAGE` of `CookiesInvalid` says that this can happen if the cookies are expired, but how would expired cookies result in `cj` being falsy?